assessments/iStation/earthmover.yml

```diff
@@ -110,13 +110,35 @@ transformations:
           - $transformations.scores_{{month}}
           {% endfor %}
 
+  # The export data is already stacked
+  renamed_export:
```

Comment:
we'll want to test this on the _other_ version this was originally written for if we haven't already. We may want a conditional around the transformations instead of around the source below. If the columns you're renaming below, for example, don't exist in the original file, then it will error. Just want to ensure backwards compatibility.

Reply:
Updated as per spec and tests with both endpoints. 

In the new case (the export endpoint): 
```
>>>> earthmover run -c earthmover.ym
l -p '{ 
> "BUNDLE_DIR": "./",
> "OUTPUT_DIR": "./short_export_out", 
> "INPUT_FILE": "./short_export.csv",
> "STATE_FILE": "state.txt",
> "STUDENT_ID_XWALK": "seeds/map_student_id.csv",
> "PERCENTILE_MAPPING": "",
> "SUBJECT": "reading",
> "ISTATION_ENDPOINT": "export"
> }'
2024-08-08 09:03:04.673 earthmover INFO starting...
2024-08-08 09:03:04.715 earthmover INFO computing input hashes for run log at state.txt
2024-08-08 09:03:04.785 earthmover INFO checking for prior runs...
2024-08-08 09:03:04.788 earthmover INFO regenerating (no prior runs found, or config.yaml has changed since last run)
2024-08-08 09:03:04.788 earthmover INFO creating output directory ./short_export_out
2024-08-08 09:03:05.377 earthmover INFO done!
```

And in the default case:
```
>>>> earthmover run -c earthmover.ym
l -p '{
> "BUNDLE_DIR": "./",
> "OUTPUT_DIR": "./short_report_out",
> "INPUT_FILE": "./short_report.csv",
> "STATE_FILE": "state.txt",
> "STUDENT_ID_XWALK": "seeds/map_student_id.csv",
> "PERCENTILE_MAPPING": "",
> "SUBJECT": "reading"
> }'
2024-08-08 09:04:08.033 earthmover INFO starting...
2024-08-08 09:04:08.093 earthmover INFO computing input hashes for run log at state.txt
2024-08-08 09:04:08.133 earthmover INFO checking for prior runs...
2024-08-08 09:04:08.135 earthmover INFO regenerating (no prior runs found, or config.yaml has changed since last run)
2024-08-08 09:04:08.136 earthmover INFO creating output directory ./short_report_out
2024-08-08 09:04:10.278 earthmover INFO done!

```

Reply:
Working on merging the fork.

Reply:
Looks like it is working